.github/components.yml

```diff
@@ -54,24 +54,16 @@ NPU:
 
 HETERO:
   revalidate:
-    - CPU
     - GPU
     - HETERO
-    - AUTO_BATCH
-    - TEMPLATE
-    - C_API
```

Comment:
@riverlijunjie Does C_API test with AUTO/HETERO/BATCH/MULTI/TEMPLATE plugins?

Reply:
CAPI test is for API level tests, so it will test some properties for AUTO and BATCH.